src/components/blocks/documentsBlock/DocumentsBlock.tsx

```diff
@@ -56,7 +56,9 @@ export const DocumentsBlock = ({ countries, family }: IProps) => {
         )}
 
         {/* Table */}
-        {view === "table" && <InteractiveTable<TEventTableColumnId> columns={tableColumns} rows={tableRows} />}
+        {view === "table" && (
+          <InteractiveTable<TEventTableColumnId> columns={tableColumns} rows={tableRows} defaultSort={{ column: "date", ascending: false }} />
```

Comment:
note to self - [these column ids are generated here](https://github.com/climatepolicyradar/navigator-frontend/blob/7305a9e951b4123cb207fc99c009c818805d14dc/src/utils/eventTable.tsx#L22)